validation/tdd_latest.xsd

```diff
@@ -209,12 +209,16 @@ as passed-in parameters.
     <xs:restriction base="xs:string">
       <xs:enumeration value="Case"/>
       <xs:enumeration value="BalancedIIF"/>
+      <xs:enumeration value="BalancedIIF_"/>  <!-- added 2021.4 -->
+      <xs:enumeration value="BalancedIF"/>  <!-- added 2021.4 -->
     </xs:restriction>
   </xs:simpleType>
   <xs:simpleType name="IfThenElseApproach-ST">
     <xs:restriction base="xs:string">
       <xs:enumeration value="Case"/>
       <xs:enumeration value="IIF"/>
+      <xs:enumeration value="IIF_"/>
```

Comment:
Can you add the comments from 2021.3 as well here.  As long as you keep those lines identical it'll make the merge to/from master a bit easier.

Reply:
Done, thanks!